src/metrics/migrations/0006_auto_20190627_1412.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import re
+
+from django.db import migrations
+
+from metrics.logic import get_iso_country_code
+
+
+def process_accesses(apps, schema_editor):
+    ArticleAccess = apps.get_model('metrics', 'ArticleAccess')
+    Country = apps.get_model('core', 'Country')
+
+    accesses = ArticleAccess.objects.all()
+
+    for access in accesses:
+        print(access.pk)
+        check = re.match(
+            r"^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$",
+            access.identifier,
+        )
+
+        if check:
+            print('ip match')
```

Comment:
I suppose we don't need these print statements

Reply:
Indeed, removed.